docs/sources/configure-client/trace-span-profiles/java-span-profiles.md

```diff
@@ -40,49 +40,52 @@ Your applications must be instrumented for profiling and tracing before you can
 
 ## Configure the otel-profiling-java package
 
-To start collecting Span Profiles for your Java application, you need to include [otel-profiling-java](https://github.com/pyroscope-io/otel-profiling-java) in your code.
+To start collecting Span Profiles for your Java application, you need to include [otel-profiling-java](https://github.com/pyroscope-io/otel-profiling-java) as an extension to your application.
 
-This package is a `TracerProvider` implementation, that labels profiling data with span IDs which makes it possible to query for span-specific profiling data in Grafana Tempo UI.
+Let's assume you have this sample application docker image:
 
-```shell
-java -jar ./build/libs/rideshare-1.0-SNAPSHOT.jar \
-    -javaagent:./opentelemetry-javaagent.jar \
-    -Dotel.javaagent.extensions=./pyroscope-otel.jar \
-    -Dotel.pyroscope.start.profiling=true \
-    -Dpyroscope.application.name=ride-sharing-app-java-instrumentation  \
-    -Dpyroscope.format=jfr \
-    -Dpyroscope.profiler.event=itimer \
-    -Dpyroscope.server.address=$PYROSCOPE_SERVER_ADDRESS \
-    # rest of your otel-java-instrumentation configuration
+```Dockerfile
+# [...]
 
+EXPOSE 5000
+
+CMD ["java", "-Dserver.port=5000", "-jar", "./my-app.jar" ]
 ```
 
-Next, you need to create and configure the tracer provider:
-```java
-implementation("io.pyroscope:otel:0.10.1.11")
+Now by adding the OTEL Java agent and our Pyroscope Otel Java Agent extensions, you can enrich your profiles with span IDs which makes it possible to query for span-specific profiling data in Grafana Tempo UI:
```

Comment:
Some consistent suggestions. I removed the UI reference for Tempo since Tempo doesn't have a UI per se, not the way that Pyroscope does. 
```suggestion
By adding the OTel Java agent and the Pyroscope OTel Java Agent extensions, you can enrich your profiles with span IDs which makes it possible to query for span-specific profiling data in Grafana Tempo:
```